packages/components/demo/js/components/ComponentExample/component-overrides.scss

```diff
@@ -151,20 +151,15 @@
     margin-right: 0;
   }
 
-  & .tabs {
-    & > div {
+  .tabs {
+    > div {
       min-width: 100%;
 
       @include breakpoint('42rem') {
         min-width: 720px;
       }
     }
 
-    .bx--tabs + div {
```

Comment:
Would we want to keep this in but add a note to remove in v11? This seems like it could break certain layouts that may rely on this behavior currently.

Reply:
this should be demo only code, unless you are referring to something else?